gfx-memory/fuzz/fuzz_targets/dedicated.rs

```diff
@@ -0,0 +1,30 @@
+#![no_main]
+
+use gfx_backend_empty::Backend;
+use gfx_memory::{Allocator, DedicatedAllocator, DedicatedBlock};
+use gfx_memory_fuzz::{create_device, Allocation};
+use hal::{memory::Properties as MemoryProperties, MemoryTypeId};
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|allocations: Vec<Allocation>| {
```

Comment:
The fuzzing strategy currently is to generate a random number of allocations of random size and alignment, allocate them in order, then deallocate them all